listenbrainz_spark/stats/user.py

```diff
@@ -17,8 +17,12 @@ def get_artists(table):
         table: name of the temporary table
 
     Returns:
-        artists: list containing information of artists 
-                grouped by users
+        artists: A dict of dicts containing artists information of every user 
```

Comment:
This description is good, but the function still needs a line in the beginning of the docstring explaining what it does.

we follow the format mentioned here: https://github.com/metabrainz/guidelines/blob/master/Python.md#docstrings

Consistency is important as it makes things easier to understand. :)

Reply:
yeah. right. Thanks for the link :)